.bazelignore

```diff
@@ -0,0 +1,2 @@
+e2e/node_package
```

Comment:
I've instead followed the pattern of listing the nested workspaces in the root WORKSPACE to prevent traversing into them. Not sure which is better, maybe they're equivalent. Should we be consistent?

Reply:
I prefer here myself so WORKSPACE is cleaner